CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
   This module is unstable and breaking changes may be introduced.
   See our [versioning policy](VERSIONING.md) for more information about these stability guarantees. (#5629)
 
+### Changed
+
+- `NewMemberRaw`, `NewKeyProperty` and `NewKeyValuePropertyRaw` in `go.opentelemetry.io/otel/baggage` allow UTF-8 string in key. (#5132)
```

Comment:
```suggestion
- `NewMember`, `NewMemberRaw`, `NewKeyProperty` and `NewKeyValuePropertyRaw` in `go.opentelemetry.io/otel/baggage` allow UTF-8 string in key. (#5132)
```

Reply:
It is quite strange to me that the value is percent-encoded, but the key is UTF-8. And, I thought we want to keep `NewMember` is behavior intact. https://github.com/open-telemetry/opentelemetry-go/pull/5132#discussion_r1680415740

Reply:
You are right. Resolving